ledger/participant-integration-api/src/test/lib/scala/platform/store/platform/partymanagement/PartyRecordStoreTests.scala

```diff
@@ -293,6 +307,53 @@ trait PartyRecordStoreTests extends PartyRecordStoreSpecBase { self: AsyncFreeSp
         }
       }
     }
+
+    "raise an error when annotations byte size max size exceeded" - {
```

Comment:
Missing tests for annotations invalid in other ways defined in `validateAnnotations`

Reply:
I'm assuming that annotations invalid in other ways will not go through the store layer. Rather, they will be rejected at the API service layer. 

To better reflect this in the code, I'm thinking about introducing a type to represent valid annotation key e.g. 'type AnnotationKey' similar to how we use 'Ref.Party' or 'Ref.UserId'. I'd do it in a separate ticket after the api service layer